### DISCUSSION TITLE
uYouPlus 17.35.3

### DISCUSSION THREAD

**Discussion by risingkazuya05** [qnblackcat/uYouPlus#442] [General]
With the latest update, to uYouPlus 17.35.3, it says in the change log that if installed using TrollStore, it will use the official VP9 codec versus the other version that leads to stuttering, etc. That being said, is there still a need to enable the VP9 codec when installing via AltStore, Sideloadly etc. to get anything higher than 1080p resolution? I have run the same videos with VP9 enabled and then disabled (restarting app each time) and the same video quality settings show 2160p and higher when the feature is either on or off. Thanks

**Comment by PoomSmart** [comment]
Yep, it's not needed. It's for non-TrollStore users more like at this point.

**Reply by risingkazuya05** [reply]
Great, thanks for confirming. Sad to lose TrollStore whenever I update to iOS16 but glad that uYouPlus is still working :)
